src/potentials.h

```diff
@@ -396,29 +397,28 @@ struct RepulsionR3 : public PairPotentialBase {
 
 /**
  * @brief Hertz potential
- * @details This is a repulsive potential describes the change in elasticenergy of two deformable objects when subjected
- * to an axialcompression.
+ * @details This is a repulsive potential, that for example describes the change in elastic energy
+ * of two deformable objects when subjected to an axial compression.
```

Comment:
rename R_h to sigma and remove any mention of "hydrodynamic"

Reply:
I thought about it and R_H could also mean a Hertz radius, cf. used eps_H. To stay consistent with `AtomData` I would keep the naming scheme as it is so far. However I already have a plan how to refactorize `AtomData` and pair potentials together with names changes.

Reply:
OK, but I would like "hydrodynamic" to be eradicated. It should never have been there in the first place.

Reply:
Kill it before it breed! :-D I amended the last commit and force-pushed. Not even a single mention about hydrodynamics. ;-)